Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -90,6 +91,36 @@ lemma enorm_eq_nnnorm (x : E) : ‖x‖ₑ = ‖x‖₊ := rfl
 
 end ENorm
 
+/-- A type `E` equipped with a continuous map `‖·‖ₑ : E → ℝ≥0∞`. -/
+class ContinuousENorm (E : Type*) [TopologicalSpace E] extends ENorm E where
+  continuous_enorm : Continuous enorm
+  -- Future: should one demand that the topology is somehow defined by the enorm?
+  -- For many applications, the current definition suffices.
```

Comment:
probably not, because that's not true for ennreal? Maybe add a comment about that?

Reply:
Good idea! Added a comment.